### PR TITLE
NOT FOR MERGING

### PR DESCRIPTION
**Pull request by tpoisot**
This is just to get some very quick feedback -- is the section 2 long enough, and is it OK if the next sections are ~ of the same size? Pinging @gvwilson

### CODE REVIEW

File: 02-basics.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+~~~ md
+We can do `python`:
+
+``` python
+for i in xrange(5):
+  print "This is line " + str(i) + " of this useless loop.\n"
+```
+
+Easy peasy!
```
Comment (gvwilson) [inline 02-basics.md]: We try to avoid saying "this is easy" because for some people it isn't (cf. not using the word "just").

**Comment by gvwilson** [comment]
Looks good to me - I would add sections in 02-basics on math & 
equations, tables, and figures, but this is on the right track. What 
exercises will you set at the end of each section?

**Comment by tpoisot** [comment]
I added bits on the tables, references, figures, ...

How many exercices should I plan for?

**Comment by gvwilson** [comment]
I would turn it around - as discussed in instructor training, I'd figure 
out the exercises (formative assessments) and then fill in the lessons 
underneath them.  What do you expect learners to be able to do once 
they're done this lesson? What skills will they have, what problems will 
they be able to solve, or what understanding of abstract concepts will 
they be able to demonstrate (and how)?
Thanks,
Greg

**Comment by tpoisot** [comment]
Good advice. I'll work on this.

**Pull request merged**